jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -606,6 +614,12 @@ def calculate_photometry(self, dataset=None, aperture=None, background=None,
             # we can use the pre-cached value
             data = self.dataset.selected_dc_item
 
+        if self.config == "cubeviz" and data.ndim > 2:
+            if "spectral_axis_index" in getattr(data, "meta", {}):
+                spectral_axis_index = data.meta["spectral_axis_index"]
+            else:

```

Comment:
when would case occur, where "spectral_axis_index" is not in meta and it defaults in this plugin to 0?